### PR TITLE
Added Option to filter in LoggedBlocks List (GUI)

### PR DESCRIPTION
**Pull request by borekking**

### CODE REVIEW

File: src/de/cuuky/varo/gui/admin/orelogger/OreLoggerFilterGUI.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        addItem(20 + 9, new BuildItem().material(Material.REDSTONE).displayName(ChatColor.RED + "Reset Player").build(),
+                this.playerFilter.resetFilter(this.getPlayer())
+        );
+
+        // Filter material
+        addItem(24, new BuildItem().material(Material.SIGN).displayName(Main.getColorCode() + "Filter Material" + ChatColor.GRAY + " (" + this.materialFilter.getContent() + ")").build(),
+                this.materialFilter.setFilter(this.getPlayer(), Main.getPrefix() + "Please enter a material name to filter for:"));
+
+        // Reset material
+        addItem(24 + 9, new BuildItem().material(Material.REDSTONE).displayName(Main.getColorCode() + "Reset Material").build(),
```
Comment (Almighty-Satan) [inline src/de/cuuky/varo/gui/admin/orelogger/OreLoggerFilterGUI.java:105]:
```suggestion
        addItem(24 + 9, new BuildItem().material(Material.REDSTONE).displayName(ChatColor.RED + "Reset Material").build(),
```

**Review by Almighty-Satan — CHANGES_REQUESTED** [review]

**Comment by UeberallGebannt** [comment]
Vorschlag: LanguageSystem benutzen statt Nachrichten zu hardcoden

**Comment by Almighty-Satan** [comment]
The user should also get an error message when attempting to sort by an invalid player or material

**Comment by CuukyOfficial** [comment]
Please use Materials.* instead of Material.* in every BuildItem#material call. Thanks. :)

**Comment by UeberallGebannt** [comment]
Please use the replace() method instead of replaceAll() if you don't want to use regex

**Pull request merged**